src/plugins/data_explorer/public/application.tsx

```diff
@@ -5,32 +5,33 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider as ReduxProvider } from 'react-redux';
 import { Router, Route, Switch } from 'react-router-dom';
 import { AppMountParameters, CoreStart } from '../../../core/public';
 import { OpenSearchDashboardsContextProvider } from '../../opensearch_dashboards_react/public';
 import { DataExplorerServices } from './types';
 import { DataExplorerApp } from './components/app';
+import { Store } from './utils/state_management';
 
 export const renderApp = (
-  { notifications, http }: CoreStart,
+  core: CoreStart,
```

Comment:
`core` and `params` appear unused?

Reply:
params is passed on to the views since they need it to render the top nav menu items correctly. core is not used right now but will be needed later on.